web/modules/custom/itk_cookie_message/src/Plugin/Block/ITKCookieMessage.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * @file
+ * Contains ITKCookieMessage block.
+ */
+
+namespace Drupal\itk_cookie_message\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides cookie message block.
+ *
+ * @Block(
+ *   id = "itk_cookie_message",
+ *   admin_label = @Translation("ITK cookie message"),
+ * )
+ */
+class ITKCookieMessage extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $settings = _itk_cookie_message_get_settings();
+    $config = \Drupal::config('itk_cookie_message.settings');
+
+    $cookieName = $config->get('cookie_name');
+    $cookieLifetime = $config->get('cookie_lifetime');
+
+    return [
+      '#type' => 'markup',
+      '#theme' => 'itk_cookie_message_block',
+      '#variables' => [
```

Comment:
These default "strings" should not be necessary if the default config trick have been used - mentioned before: https://www.drupal.org/docs/8/creating-custom-modules/add-a-default-configuration